src/Analyzer.Cli/ExitCode.cs

```diff
@@ -14,9 +14,10 @@ public enum ExitCode
         Success = 0,
 
         /// <summary>
-        /// Error: GenericError
+        /// Error: There was an error with the command.
+        /// This is returned by rootCommand.InvokeAsync in the CLI if there is an error.
         /// </summary>
-        ErrorGeneric = 1,
+        ErrorCommand = 1,
```

Comment:
Please update the ExitCodes in /Readme.md (line ~68)

Reply:
Yes, that was my plan. :) (https://github.com/Azure/template-analyzer/pull/245#discussion_r893973699)